### DISCUSSION TITLE
预设皮肤及披风（仿官启）

### DISCUSSION THREAD

**Discussion by HansHG-1016** [Meloong-Git/PCL#4346] [讨论]
像官方启动器那样，可以预设多套皮肤（及披风），图片文件保存在:/PCL      文件夹下，配合3D预览使用

**Comment by MoYuan-CN** [comment]
你如果需要这个的话，很不好意思：#343 本地皮肤库，已被拒绝
额外信息：3D 皮肤预览 #945 -> #950
